### PR TITLE
CWI-216 - Messaging Extension - Update the endpoint in SDK to support the new date parameters

### PR DESCRIPTION
**Pull request by KatieHendricksen**
We needed to update the SDK based on this PR: https://github.com/tactivos/mural-api/pull/10066/files

I tested it locally and it works!

[Jira Ticket](https://mural.atlassian.net/browse/CWI-216)

### CODE REVIEW

**Review by KavenTheriault — CHANGES_REQUESTED** [review]
Don't forget to update the `searchCrossWorkspaceMurals` function implementation at line 677

File: packages/mural-client/src/index.ts [hunk 1]
```diff
@@ -167,7 +167,39 @@ const optionsParams = (
   return params;
 };
 
-// ====
+const buildCrossWorkspaceParams = (q:string, createdSince?: number | undefined, createdUntil?: number | undefined, createdFilterStrict?: boolean | undefined, updatedSince?: number | undefined, updatedUntil?: number | undefined, updatedFilterStrict?: boolean | undefined) => {
+  const params = new URLSearchParams();
+
+  if(q){
```
Comment (KavenTheriault) [inline packages/mural-client/src/index.ts:173]: `q` is mandatory, we can remove this `if`

File: packages/mural-client/src/index.ts [hunk 2]
```diff
@@ -167,7 +167,39 @@ const optionsParams = (
   return params;
 };
 
-// ====
+const buildCrossWorkspaceParams = (q:string, createdSince?: number | undefined, createdUntil?: number | undefined, createdFilterStrict?: boolean | undefined, updatedSince?: number | undefined, updatedUntil?: number | undefined, updatedFilterStrict?: boolean | undefined) => {
```
Comment (KavenTheriault) [inline packages/mural-client/src/index.ts:170]: By using question mark at the end of the parameter name, `| undefined` is not needed

File: packages/mural-client/src/index.ts [hunk 3]
```diff
@@ -667,9 +706,17 @@ const buildApiClient = (
...
+      createdSince,
+      createdUntil,
+      createdFilterStrict,
+      updatedSince,
+      updatedUntil,
+      updatedFilterStrict }, options) => {
       const params = optionsParams(options);
-      const response = await fetchFn(api(`search/mural?q=${q}${params}`), {
+      const crossWorkspaceTimeParams = buildCrossWorkspaceParams(q, createdSince, createdUntil, createdFilterStrict, updatedSince, updatedUntil, updatedFilterStrict);
+      const response = await fetchFn(api(`search/mural?${crossWorkspaceTimeParams}?${params}`), {
```
Comment (KavenTheriault) [inline packages/mural-client/src/index.ts:719]: I think the second `?` should be removed

Comment (KatieHendricksen) [reply packages/mural-client/src/index.ts]: Sounds good!

**Review by KavenTheriault — APPROVED** [review]
LGTM

**Pull request merged**